zero_bin/common/src/block_interval.rs

```diff
@@ -7,8 +10,13 @@ use futures::Stream;
 use tracing::info;
 
 use crate::parsing;
+use crate::provider::CachedProvider;
+
+// Time in milliseconds to poll the node for new blocks.
+// We set it to 12 seconds, default ETH block time.
+const DEFAULT_BLOCK_TIME: u64 = 12000;
 
-const DEFAULT_BLOCK_TIME: u64 = 1000;
+pub type BlockIntervalStream = Pin<Box<dyn Stream<Item = Result<(u64, bool), anyhow::Error>>>>;
```

Comment:
Maybe copy-paste the type explanation from below here as well, so it's clear what the result corresponds to.